.changeset/gentle-cows-remain.md

```diff
@@ -0,0 +1,13 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Accordion`: Add `@titleTag` argument
+
+`Alert`: Add `@tag` argument to `[A].Title`
+
+`ApplicationState`: Update the `@titleTag` argument to only accept `"div" | "h1" | "h2" | "h3" | "h4" | "h5" | "h6"`
+
+`CodeBlock`: Add `@tag` argument to `[CB].Title`
+
+`DialogPrimitive`: Add `@titleTag` argument to `DialogPrimitive::Header`
```

Comment:
I suggest we also add `Modal` and `Flyout` here, as consumers may not be familiar with the composition

Reply:
Hmm.. we could, but I think in the next section of work we should hard code the value for modal and flyout to be H1.

Reply:
ah, right! then this should be enough for now